home/models/models.py

```diff
@@ -47,6 +41,9 @@ class Classroom(mdls.Model):
     )
     volunteers = mdls.ManyToManyField(DjangoUser, related_name="classroom_volunteers")
     students = mdls.ManyToManyField(DjangoUser, related_name="classroom_students")
```

Comment:
The teacher, teacher_assistant, volunteers, and students could all be unified with a many to many field through another table that had an enum for what kind of relationship they have with the classroom.  https://docs.djangoproject.com/en/2.2/topics/db/models/#extra-fields-on-many-to-many-relationships

Reply:
This is a big one, but I think it will make it a lot easier to add new parameters to members of a class down the line so I am going to implement it.

Reply:
Done in 683281caa50454ef27887fd938e7d2749a76cbcc, f7e8dc400f01753d2d56978456a6dc5030f2f57e, ad44129218385500743a338b0f6d647982df3713